tox.ini

```diff
@@ -83,7 +88,12 @@ deps =
 [testenv:pinned]
 deps =
     {[pinned]deps}
-    lxml==3.5.0
+    PyDispatcher==2.0.5
+
+[testenv: pypy-pinned]
+deps = 
+    {[pinned]deps}
+    PyPyDispatcher==2.1.0
```

Comment:
This testenv should be removed, and instead `pypy3-pinned` modified accordingly (or rather left as it was).